Cargo.toml

```diff
@@ -3,16 +3,24 @@ name = "monolith"
 version = "2.0.9"
 authors = [
     "Sunshine <sunshine@uberspace.net>",
-    "Mahdi Robatipoor <mahdi.robatipoor@gmil.com>",
+    "Mahdi Robatipoor <mahdi.robatipoor@gmail.com>",
 ]
 description = "CLI tool for saving web pages as a single HTML file"
+homepage = "https://github.com/Y2Z/monolith"
+keywords = ["commandline","HTML","download"]
+license = "UNLICENSE"
+repository = "https://github.com/Y2Z/monolith"
 
 [dependencies]
 base64 = "0.10.1"
-clap = "2.33.0"
 html5ever = "0.24.0"
 indicatif = "0.11.0"
 lazy_static = "1.3.0"
 regex = "1.2.1"
 reqwest = "0.9.20"
 url = "2.1.0"
+structopt = "0.2.18"
+
+
+
+
```

Comment:
way too many newlines Ü

I'm not sure what the advantage of structopt over clap would be, it seems to be a less popular package. Perhaps we should have a separate PR for it, as it seems to be a big change for the project, I wouldn't want to hold CI/CD pipeline from checking the code because of it.